frontend/src/pages/Registration.tsx

```diff
@@ -0,0 +1,108 @@
+// tailwind docs: https://v2.tailwindcss.com/docs/
+import React, { useState } from "react";
+import axios from "axios";
+
+const API_BASE_URL = "http://localhost:5001/api/profiles";
+
+const UserRegistration: React.FC = () => {
+    const [formData, setFormData] = useState({
+        firstName: "",
+        lastName: "",
+        email: "",
+        phone: "",
+        institution: "",
+        fieldOfInterest: "",
+        bio: "",
+        password: "",
+        confirmPassword: ""
```

Comment:
you will need to add a userId field for this, including a field where they can choose a unique username. this is so that it will work with the profile pages.